modules/imaging_browser/php/imaging_browser.class.inc

```diff
@@ -112,10 +112,12 @@ class Imaging_Browser extends \DataFrameworkMenu
         }
 
         // get list of projects
+        // PROJECTS
+        $allProjectList   = \Utility::getProjectList();
         $list_of_projects = array();
-        $projectList      = \Utility::getProjectList();
-        foreach ($projectList as $key => $project) {
-            unset($projectList[$key]);
+        $userProjectsIDs  = $user->getData('ProjectIDs');
```

Comment:
try $user->getProjectIDs() per dave's comment in #6831
Should be worth a new function in User to get the project list if this code needs to be recycled more than once across multiple modules

Reply:
per my comment above -- Based on your [investigation across modules](https://github.com/aces/Loris/issues/6829#issuecomment-656357582) - 
Why not add a User class function to perform these 4-5 lines of code?

Reply:
I think the foreach is formatting an array for the select filter, this can change between modules, and the snippet is doing formatting fir the select filters. I don't think it makes sense in the User class.